pycqed/analysis_v3/processing_pipeline.py

```diff
@@ -516,6 +516,8 @@ def resolve_keys_in(self, keys_in, mobj_name, meas_obj_value_names_map,
                 else:
                     raise ValueError('The first node in the pipeline cannot '
                                      'have "keys_in" = "previous".')
+            else:
+                keys_in += [keyi]
```

Comment:
Am I getting it wrong or is this the same action as in the first branch of the if? If yes, the code could be simplified. It might also be useful to add a comment why this is the default behavior.

Reply:
Good catch! I now removed the first if and kept the else. Plus added a comment under the else